debian/rules

```diff
@@ -40,63 +40,63 @@ override_dh_install:
-		"176065564a9979f37388b87fb88b57dfaa77e7fee8885030a48f81ab1d265e92" \
+		"hpux/jdk1.8/jdk-8.0.23-hpux-ia64.tar.gz" \
+		"18c4aa183b551374a7b22a8eda3427d791a4869d3d797a61ae22bcc20451852d" \
 		"debian/tmp/usr/share/host-jdks/jdk/hpux_ia64/jdk1.8/jdk.tar.gz"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"hpux/jdk1.8/jdk-8.0.22-hpux-ia64-manifest" \
-		"3901d4f3b1f8047978dfc5fc97bfb81ca7a2df3675252334bb8338a32385ac9c" \
+		"hpux/jdk1.8/jdk-8.0.23-hpux-ia64-manifest" \
+		"85e8bedda0e58310b31c038333546c902c5e8d58d169a385aa607647e281ef5d" \
 		"debian/tmp/usr/share/host-jdks/jdk/hpux_ia64/jdk1.8/manifest"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"sunos/jdk1.8/OpenJDK8U-jdk_x64_solaris_hotspot_8u282b08.tar.gz" \
-		"90b08b369e2803540cca4857b72bb7e72a8a304f7d67b78bf38fd28adcd9dd24" \
+		"sunos/jdk1.8/OpenJDK8U-jdk_x64_solaris_hotspot_8u322b06.tar.gz" \
+		"a9097e7e1d160d20f55db5ff1863c295ba6823e111b3bc069ac881022d9b4423" \
 		"debian/tmp/usr/share/host-jdks/jdk/sunos_x86/jdk1.8/jdk.tar.gz"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"sunos/jdk1.8/OpenJDK8U-jdk_x64_solaris_hotspot_8u282b08-manifest" \
-		"10daf9c2f73857e9e0f20c8ca3a2d4d7258ea9a6667cc1caae94489489bd81c4" \
+		"sunos/jdk1.8/OpenJDK8U-jdk_x64_solaris_hotspot_8u322b06-manifest" \
+		"8ea3fec5cb16a99bff5522c3aecd8c2d398cbe4f15541ee6b0c579566321282c" \
 		"debian/tmp/usr/share/host-jdks/jdk/sunos_x86/jdk1.8/manifest"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"sunos/jdk1.8/OpenJDK8U-jdk_sparcv9_solaris_hotspot_8u282b08.tar.gz" \
-		"e5117c387635d04ff4a575436ce4844a327653d047a1bbafc2bde57083554176" \
+		"sunos/jdk1.8/OpenJDK8U-jdk_sparcv9_solaris_hotspot_8u322b06.tar.gz" \
+		"2f7b4d9495748bb3ebf5311b0cb4fbc06508b0d3ead7064ae6804d52d63a3ef9" \
 		"debian/tmp/usr/share/host-jdks/jdk/sunos_sparc/jdk1.8/jdk.tar.gz"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"sunos/jdk1.8/OpenJDK8U-jdk_sparcv9_solaris_hotspot_8u282b08-manifest" \
-		"66c766649a6dcfa85ad3e43421bf874700adea769268f615d5a9f2a0e02a3d12" \
+		"sunos/jdk1.8/OpenJDK8U-jdk_sparcv9_solaris_hotspot_8u322b06-manifest" \
+		"4d6d8ad850b66c1b87966e5e61cd46e6e43042070ab6c97345a7f031d6e1aaca" \
 		"debian/tmp/usr/share/host-jdks/jdk/sunos_sparc/jdk1.8/manifest"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"windows/jdk1.8/OpenJDK8U-jdk_x64_windows_hotspot_8u302b08.zip" \
-		"9304c5d8ef638eafc641c6b426af87e1eea2c28fc2e64857e1307453238f587d" \
+		"windows/jdk1.8/OpenJDK8U-jdk_x64_windows_hotspot_8u322b06.zip" \
+		"456ca02a8f87ccb7dfc17b7d67c47c1768828898810e05650f78f9e88a5eda0d" \
 		"debian/tmp/usr/share/host-jdks/jdk/windows_x86/jdk1.8/jdk.zip"
 
 	./scripts/fetch-file-from-artifactory.sh \
-		"windows/jdk1.8/OpenJDK8U-jdk_x64_windows_hotspot_8u302b08-manifest" \
-		"f86a4b1ae10b02f3a1e52f7ed735604f58d372c3f8a0ff271032d8e17a50a11e" \
+		"windows/jdk1.8/OpenJDK8U-jdk_x64_windows_hotspot_8u322b06-manifest" \
+		"b2f1ff7260d6f43446dc2aed147bea3c2895f8a8241ede874ae282717d3e2eed" \
```

Comment:
If your time constraints allow for it, you could wait at least for the Windows release of 8u332 (seems to be a top priority and close to be ready according to https://github.com/adoptium/adoptium/issues/140).
It might even be a necessity to wait due to known vulnerabilities in 8u322; see the 8 column in the matrix at https://openjdk.java.net/groups/vulnerability/advisories/2022-04-19 .

Reply:
Adopt was supposed to release the 332 version on April 19th initially and We were supposed to get these in Before end of April. So, we were kind of waiting for the binaries to be available. Delaying it further might push out upgrading JDKs to another release. We can assess the vulnerabilities and make a decision based on it I guess.

Reply:
Also, When taking a quick look at the vulnerabilities mentioned in the list, All of them are undergoing reanalysis and seem to me mainly effecting Oracle JDK.

Reply:
Yes. My main concern is that customers tend to flag vulnerable software even when we are not affected in particular.